marimo/_sql/engines.py

```diff
@@ -76,19 +81,39 @@ def dialect(self) -> str:
     def execute(self, query: str) -> Any:
         # Can't use polars.imported() because this is the first time we
         # might come across polars.
+        if not (
+            DependencyManager.polars.has() or DependencyManager.pandas.has()
+        ):
+            raise_df_import_error("polars")
+
+        if DependencyManager.sqlalchemy.has():
```

Comment:
would this ever return false? i think this class is only created if sqlalchemy exists.